app/mesh/1.2.x/installation/openshift.md

```diff
@@ -231,46 +222,44 @@ $ kumactl config control-planes add --name=XYZ --address=http://{address-to-kong
 {% endnavtab %}
 {% endnavtabs %}
 
-You will notice that {{site.mesh_product_name}} automatically creates a `Mesh`
+Notice that {{site.mesh_product_name}} automatically creates a `Mesh`
 entity with the name `default`.
 
-<div class="alert alert-ee blue">
-<strong>Note:</strong> {{site.mesh_product_name}} explicitly specifies a UID
-for <code>kuma-dp</code> sidecar to avoid capturing traffic from
-<code>kuma-dp</code> itself. For that reason, a <code>nonroot</code>
-<a href="https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html">Security Context Constraint</a>
-has to be granted to the application namespace:
+{{site.mesh_product_name}} explicitly specifies a UID
+for the `kuma-dp` sidecar to avoid capturing traffic from
+`kuma-dp` itself. You must grant a `nonroot` [Security Context Constraint]
+(https://docs.openshift.com/container-platform/latest/authentication/managing-security-context-constraints.html)
+to the application namespace:
 
-<pre>
-<code>$ oc adm policy add-scc-to-group nonroot system:serviceaccounts:&lt;app-namespace&gt;</code></pre>
+```sh
+$ oc adm policy add-scc-to-group nonroot system:serviceaccounts:&lt;app-namespace&gt;
```

Comment:
```suggestion
$ oc adm policy add-scc-to-group nonroot system:serviceaccounts:<app-namespace>
```
The HTML entity names for the `<` and `>` characters don't render in fenced codeblocks.